pytest_doctestplus/plugin.py

```diff
@@ -21,6 +23,8 @@
     def indent(text, prefix):
         return '\n'.join([prefix + line for line in text.splitlines()])
 
+PYTEST_GT_5 = Version(pytest.__version__) > Version('5')
```

Comment:
`Version(pytest.__version__) < Version('6.0.0')` didn't work because CI pulled `6.0.0rc<blahblah>` and that evaluates to `True` even if I wanted it to be `False`.

Reply:
why not LooseVersion, that should return `False` for those (though one could argue False is the wrong answer, but in these scenarios it's the right answer we need)

Reply:
Apparently Python org hates `distutils` now or something... Using `packaging` is consistent with astropy/astropy#10590 .

Reply:
Pytest also uses `packaging`, i.e. it's a dependency of `pytest`, so having it a dependency here is not a big deal.  It will get installed regardless.